RxMarbles/OperatorsTableViewController.swift

```diff
@@ -201,15 +124,20 @@ class OperatorsTableViewController: UITableViewController, UISearchResultsUpdati
     }
 }
 
+extension OperatorsTableViewController: UISearchResultsUpdating {
+	func updateSearchResults(for searchController: UISearchController) {
+		self.viewModel.updateSearchResults(for: searchController)
+        tableView.reloadData()
+    }
+}
+
 extension OperatorsTableViewController: UIViewControllerPreviewingDelegate {
     /// Create a previewing view controller to be shown at "Peek".
     func previewingContext(_ previewingContext: UIViewControllerPreviewing, viewControllerForLocation location: CGPoint) -> UIViewController? {
         // Obtain the index path and the cell that was pressed.
-        guard let indexPath = tableView.indexPathForRow(at: location),
-              let cell = tableView.cellForRow(at: indexPath)
-        else { return nil }
+        guard let indexPath = tableView.indexPathForRow(at: location), let cell = tableView.cellForRow(at: indexPath) else { return nil }
```

Comment:
break second let into new line
```suggestion
        guard let indexPath = tableView.indexPathForRow(at: location),
        	  let cell = tableView.cellForRow(at: indexPath) else { return nil }
```